docs/Installation/Installation.md

```diff
@@ -6,19 +6,142 @@ See LICENSE.txt for details.
 
 \tableofcontents
 
-This page details the installation procedure for SpECTRE on personal computers
-using x86-64 processors. For configuring SpECTRE, please refer to the
-\subpage spectre_build_system page. For instructions on installing SpECTRE on
-clusters please refer to the \subpage installation_on_clusters page. For
-instructions on installing SpECTRE on Apple Silicon Macs, please refer to the
-\subpage installation_on_apple_silicon page. Refer to the
-\subpage versioning_and_releases page for information on specific versions
-to install.
+This page details how to install SpECTRE on personal machines and on clusters
+that have no official support (yet).
 
-### Install a specific version with Spack
+- For details on installing SpECTRE on a number of clusters that we support
+  please refer to:
+  \subpage installation_on_clusters
+- For configuring SpECTRE please refer to:
+  \subpage spectre_build_system
+- For instructions on installing SpECTRE on Apple Silicon Macs please refer to:
+  \subpage installation_on_apple_silicon
+- For information on our versioning scheme and public releases please refer to:
+  \subpage versioning_and_releases
 
-You can install SpECTRE with the [Spack](https://github.com/spack/spack) package
-manager:
+### Running containerized releases
+
+A quick way to run the code without installing anything at all is with our
+containerized releases:
+
+```
+docker run sxscollaboration/spectre --help
+```
+
+You can also use [Apptainer/Singularity](https://apptainer.org) instead of
+Docker, which works better on computing clusters and is more convenient because
+it shares the host's file system:
+
+```
+apptainer run docker://sxscollaboration/spectre --help
+```
+
+The entrypoint to this container is the SpECTRE
+\ref tutorial_cli "command-line interface (CLI)".
+For example, you can generate initial data for a simulation of merging black
+holes and plot the result like this:
+
+```
+apptainer run docker://sxscollaboration/spectre bbh generate-id \
+  -q 1 --chi-A 0 0 0 --chi-B 0 0 0 -D 16 -w 0.015 -a 0 -o ./bbh_id
+apptainer run docker://sxscollaboration/spectre plot slice \
+  bbh_id/BbhVolume*.h5 -C 0,0,0 -n 0,0,1 -u 0,1,0 -X 24 24 \
+  -y ConformalFactor -o plot.pdf
+```
+
+The containers currently have precompiled code only for Linux x86_64 platforms,
+and only have a limited set of executables precompiled. The supported features
+available in the precompiled containers are:
+
+- Generating initial data
+- Running CCE (see \ref tutorial_cce)
+- Running Python support code with the SpECTRE CLI (see \ref tutorial_cli)
+
+### Running precompiled executables
```

Comment:
Given the wording in the previous section, this title is a bit confusing since this section is completely unrelated to the docker container. Maybe something like "Running with static binaries" ?